### DISCUSSION TITLE
Feature Request: Allow the user to replace the timeline header with a number range

### DISCUSSION THREAD

**Discussion by lusol** [namespace-ee/react-calendar-timeline#986] [Ideas]
I really like the dragging and reordering elements feature of this library but I was wondering if it was possible to replace the timeline header with just a range of numbers? So instead of dates, we could have a lower and upper bound and then we could place elements onto the draggable area. Moving / dragging the handles of the element would change the upper and lower bound states of the element. I have attached a design to illustrate my request / question.

![dragging example 3](https://user-images.githubusercontent.com/9649910/233153387-ae811560-4328-4597-9a11-e3c818799511.png)

**Comment by chaitanya-y** [comment]
Hi lusol, Are you able to achieve this?. I also want to know how to get range in header?
